eng/pipelines/runtimelab/runtimelab-post-build-steps.yml

```diff
@@ -18,7 +18,9 @@ steps:
   # Now we need to build the cross-targeting compilers, RyuJit and ILC. Likewise with packages.
 
   - ${{ if eq(parameters.archType, 'wasm') }}:
-    - script: $(Build.SourcesDirectory)/build$(scriptExt) clr.wasmjit+clr.aot -c $(buildConfigUpper) $(_officialBuildParameter) -ci
+    - script: $(Build.SourcesDirectory)/build$(scriptExt) clr.wasmjit+clr.aot -c $(buildConfigUpper) $(_officialBuildParameter) -ci -cross
+      env:
+        ROOTFS_DIR: /crossrootfs/x64
```

Comment:
I think if you switch the container in the platform definition to `linux_x64_sanitizer`, this explicit `ROOTFS_DIR` can be dropped.

Reply:
Yes, that works, thanks